src/crafting.h

```diff
@@ -69,6 +69,8 @@ struct recipe {
         bool hidden;
     };
     std::vector<bookdata_t> booksets;
+    std::set<std::string> flags;
```

Comment:
If we're going that way, it could be a `float` instead. That way it could be more granular than just BLIND_EASY/HARD.